<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigOutputAudioClass</class>
 <widget class="QWidget" name="ConfigOutputAudioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>173</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="audioBox">
     <property name="title">
      <string>Audio settings</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelAudioTitle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>No supported audio input was found</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>12</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="labelAudioCodec">
           <property name="text">
            <string>Audio codec</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSamplingRate">
           <property name="text">
            <string>Sampling rate</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="comboBitrate"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="labelBitrate">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Bitrate (kbits/s)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="audioChannelLayout"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelChannelLayout">
           <property name="text">
            <string>Channel layout</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="audioSamplingRate"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboAudioCodec"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
